t/admin/stream-routes.t

```diff
@@ -15,13 +15,42 @@
 # limitations under the License.
 #
 use t::APISIX 'no_plan';
+use Cwd qw(cwd);
 
 repeat_each(1);
 no_long_string();
 no_root_location();
 no_shuffle();
 log_level("info");
 
+my $apisix_home = $ENV{APISIX_HOME} || cwd();
+
+sub read_file($) {
+    my $infile = shift;
+    open my $in, "$apisix_home/$infile"
+        or die "cannot open $infile for reading: $!";
+    my $data = do { local $/; <$in> };
+    close $in;
+    $data;
+}
+
+my $yaml_config = read_file("conf/config.yaml");
+$yaml_config =~ s/node_listen: 9080/node_listen: 1984/;
+$yaml_config =~ s/enable_heartbeat: true/enable_heartbeat: false/;
+$yaml_config =~ s/  # stream_proxy:/  stream_proxy:\n    tcp:\n      - 9100/;
+$yaml_config =~ s/admin_key:/admin_key_useless:/;
```

Comment:
what is `admin_key_useless`?

Reply:
One way to disable admin API token

https://github.com/apache/incubator-apisix/blob/master/t/APISIX.pm#L77

Reply:
I don't know what that means in English：admin_key_useless

Reply:
how about `disable_admin_key` ?